README.md

```diff
@@ -356,53 +326,55 @@ packages_blocklist:
   - yp-tools
   - ypbind
 packages_debian:
-  - acct
-  - apparmor-profiles
-  - apparmor-utils
-  - apt-show-versions
-  - audispd-plugins
+  # - acct
+  # - apparmor-profiles
+  # - apparmor-utils
+  # - apt-show-versions
+  # - audispd-plugins
```

Comment:
Why are these packages commented out?  Note: this goes for all of the commented-out packages in this file. There should be some explanation of why.

Reply:
See 94f7b7442d3c7ad4e858678561ac7fb081005155